logger/glog/glog_file.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 // MaxSize is the maximum size of a log file in bytes.
-var MaxSize uint64 = 1024 * 1024 * 1800
+var MaxSize uint64 = 1024 * 1800
```

Comment:
Note that this is also changed, yielding now `1.04858 MB` instead of `1.88744 GB`. A gigabyte log file seemed big.